### PR TITLE
Type checking

### PR DESCRIPTION
**Pull request by petemill**
This repo could do with some type validation.

I only did ntp-sponsored-images files and files they include (utils.js etc) so far.

### CODE REVIEW

**Review by bsclifton — APPROVED** [review]
++

File: scripts/ntp-sponsored-images/package.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ const stageFiles = (locale, version, outputDir) => {
     to: version
   }
   fs.copyFileSync(originalManifestPath, outputManifestPath)
+  // @ts-ignore typescript thinks were using es import syntax and exacts replace.default.sync
```
Comment (bsclifton) [inline scripts/ntp-sponsored-images/package.js:31]: I think you meant expects?
```suggestion
  // @ts-ignore typescript thinks were using es import syntax and expects replace.default.sync
```

**Comment by bsclifton** [comment]
Changes looked OK; only npm package added was `typescript`. However, if you look at the `package-lock.json`, you'll see updates (including to `hashset-cpp`)

@petemill are you using a newer version of NPM? Curious how those other values got updated in `package-lock.json`

**Comment by petemill** [comment]
Because they are not pinned in package.json, so anytime npm install is run
then they will get updated.

**Pull request merged**